source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -60,14 +58,13 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
   yearlyFinancialOptions,
   onSave,
   onCancel,
-  onFooterSave,
-  missingFieldsError,
 }) => {
   const fiscalYearOptions = generateFiscalYearOptions();
   const { setModalContent, setDisplayModal } = useModalContext();
   const formikRef = useRef<FormikProps<CompensationRequisitionFormModel>>(null);
   const [activitiesUpdated, setActivitiesUpdated] = useState<boolean>(false);
   const [showModal, setShowModal] = useState(false);
+  const [isValid, setIsValid] = useState<boolean>(true);
```

Comment:
not sure how we still end up with state outside of containers despite my best efforts in code review.

Reply:
As I said in my comment above, this was previously in the container but Manuel suggested that in this particular case it was less messy/convoluted to have the isValid variable declared here which I agreed with as well. Before, we were passing isValid and setIsValid as props and it made things less readable. Regardless, I will make the change :)